src/Storage/Entity/ContentValuesTrait.php

```diff
@@ -646,9 +646,13 @@ protected function getTemplateFieldsContentType()
         }
 
         if ($templateFieldsConfig = $this->app['config']->get('theme/templatefields')) {
-            $template = $this->app['templatechooser']->record($this);
-            if (array_key_exists($template, $templateFieldsConfig)) {
-                return $templateFieldsConfig[$template];
+            $templateChooser = $this->app['templatechooser'];
+            /** @var \Bolt\TemplateChooser $template */
+            $templates = $templateChooser->record($this);
+            foreach ($templates as $template) {
+                if (array_key_exists($template, $templateFieldsConfig)) {
+                    return $templateFieldsConfig[$template];
+                }
```

Comment:
No buts … 
![make-it-so](https://cloud.githubusercontent.com/assets/1427081/21751603/220c948c-d5ca-11e6-94e6-0911ff9677e4.png)